detect-monero-demo/k8s/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cryptocurrency
```

Comment:
Should you just put this in the other yaml file so there doesn't need to be a `k8s` subfolder?